### PR TITLE
[Windows] TC update for api-key geolocation when app build

### PR DESCRIPTION
**Pull request by zhuyongyong**
Impacted tests(approved): new 0, update 1, delete 0
Unit test platform: Crosswalk Project for windows 19.48.495.0
Unit test result summary: pass 1, fail 0, block 0

BUG=https://crosswalk-project.org/jira/browse/XWALK-6425

### CODE REVIEW

**Comment by zhuyongyong** [comment]
@jennycao Pls review it

**Comment by jennycao** [comment]
LGTM

File: usecase/usecase-wrt-windows-tests/README.md [hunk 1]
```diff
@@ -12,5 +12,8 @@ Copyright (c) 2015 Intel Corporation.<br/>
 Except as noted, this software is licensed under BSD-3-Clause License.<br/>
 Please see the COPYING file for the BSD-3-Clause License.
 
-## Envrionment
+## Pre-conditions
+
+Copy the usecase-wrt-windows-tests/crosswalk-app-tools-keys.json file to  /Users/<name>/.crosswalk-app-tools-keys.json
```
Comment (jennycao) [inline usecase/usecase-wrt-windows-tests/README.md]: can this step change to automatic

**Comment by jennycao** [comment]
LGTM

**Pull request merged**